beta/examples/tensorflow/common/object_detection/architecture/darknet.py

```diff
@@ -0,0 +1,65 @@
+"""
+ Copyright (c) 2020 Intel Corporation
+ Licensed under the Apache License, Version 2.0 (the "License");
+ you may not use this file except in compliance with the License.
+ You may obtain a copy of the License at
+      http://www.apache.org/licenses/LICENSE-2.0
+ Unless required by applicable law or agreed to in writing, software
+ distributed under the License is distributed on an "AS IS" BASIS,
+ WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ See the License for the specific language governing permissions and
+ limitations under the License.
+"""
+
+import tensorflow as tf
+import tensorflow.keras.backend as K
+
+from beta.examples.tensorflow.common.object_detection.architecture import nn_ops
+
+class Darknet:
+    """Class to build CSPDarknet53"""
+
+    def mish(self, x):
+        return x * K.tanh(K.softplus(x))
+
+
+    def DarknetConv2D_BN_Mish(self, *args, **kwargs):
+        """Darknet Convolution2D followed by SyncBatchNormalization and Mish."""
+        no_bias_kwargs = {'use_bias': False}
+        no_bias_kwargs.update(kwargs)
+        return nn_ops.compose(
+            nn_ops.DarknetConv2D(*args, **no_bias_kwargs),
+            tf.keras.layers.experimental.SyncBatchNormalization(),
+            tf.keras.layers.Activation(self.mish))
+
+
+    def csp_resblock_body(self, x, num_filters, num_blocks, all_narrow=True):
+        '''A series of resblocks starting with a downsampling Convolution2D'''

```

Comment:
Could you please use one format for docstrings in the example code. I mean using or `"""` or `'''`.